gloo/docs/gloo_routing/virtual_services/authentication/external_auth/apikey_auth/_index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: ApiKey Auth
+weight: 1
+description: How to setup ApiKey authentication. 
+---
+
+There are a few different ways to use Gloo External Auth. In certain cases – such as protecting low-priority routes with
+secure pre-existing identifiers – it may be desirable to secure a set of routes with **ApiKey Auth**. Keep in mind 
+that your routes are only as secure as your ApiKeys; thus ApiKeys are generally better for identification and analytics.
```

Comment:
We don't really offer anything around identification or analytics, so I'm not sure why we're mentioning them here

I think the general point is that API keys should be reserved for cases where a long-lived UUID is an appropriate model, and that it is up to the user to ensure tokens are kept private and are rotated as desired.